src/components/Footer/JoinUs/JoinUs.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../../scss/global_styles.scss";
```

Comment:
I don't know about what code style is applied in this app, but usually, imports should be in single quotes